### PR TITLE
[Manual] Add manual test plan for vm-import with EFI mode and secure boot

### PR DESCRIPTION
**Pull request by TachunLin**
### What this PR does / why we need it:
This PR is used to add manual test plan according to the latest vm-import-controller enhancement issue 
https://github.com/harvester/harvester/issues/5846

### Test Scenarios
* For OpenStack:

  - BIOS based VM Migration
  - EFI Based VM Migration
  - EFI Based VM with secure boot migration:
  - Using a custom storage class

* For VMware:

  - BIOS based VM Migration
  - EFI Based VM Migration
  - EFI Based VM with secure boot migration:
  - Using a custom storage class

### Special notes for your reviewer:
Tested on the local hugo server, image can be loaded well. 

### Additional documentation or context
Related task ticket:
https://github.com/harvester/tests/issues/1509

### CODE REVIEW

File: docs/content/manual/advanced/addons/5846-vm-import-enhancement.md [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+## OpenStack Prerequisite Setup
+1. Prepare a baremetal or virtual machine to host the OpenStack service 
+1. Or use the automated Jenkins pipeline to prepare a devstack cluster (Openstack 16.2) (stable/train)
+1. Install OpenStack command line tool on your local machine (introduce in next section)
+
+## VMware vSphere Prerequisite Setup
+1. Get the available access to the existing or prepared vSphere client (v7.1)
+
+## Harvester Prerequisite Setup
+1. Harvester can connect to the OpenStack dashbord and API endpoint
```
Comment (albinsun) [inline docs/content/manual/advanced/addons/5846-vm-import-enhancement.md:19]: Typo, dashboard

File: docs/content/manual/advanced/addons/5846-vm-import-enhancement.md [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+1. Harvester can connect to the vShpere client dashboard and API endpoint
```
Comment (albinsun) [inline docs/content/manual/advanced/addons/5846-vm-import-enhancement.md:20]: typo, vSphere.

File: docs/content/manual/advanced/addons/5846-vm-import-enhancement.md [hunk 3]
```diff
@@ -0,0 +1,453 @@
...
+   - Click the `Download OpenStack RC File` button
+   - Select the `OpenStack cloud.yaml File`
+1. Copy the cloud.yaml file to your openstack config folder 
+  ```
+  mkdir -p ~/.config/openstack
+  cp -v ~/Documents/Harvester/Openstack/clouds.yaml ~/.config/openstack
+  ```
+1. Check can list the image list on OpenStack Server 
+  ```
+  openstack image list --os-auth-url http://172.19.109.182/identity --os-identity-api-version 3 --os-project-name admin --os-project-domain-name default --os-username <username> --os-password <password>
```
Comment (albinsun) [inline docs/content/manual/advanced/addons/5846-vm-import-enhancement.md:46]: Avoid IP exposing?

File: docs/content/manual/advanced/addons/5846-vm-import-enhancement.md (hunk 1 above)
Comment (TachunLin) [reply docs/content/manual/advanced/addons/5846-vm-import-enhancement.md:19]: Thanks for the check. Update the typos to dashboard.

File: docs/content/manual/advanced/addons/5846-vm-import-enhancement.md (hunk 2 above)
Comment (TachunLin) [reply docs/content/manual/advanced/addons/5846-vm-import-enhancement.md:20]: Thanks for the check. Update the typos to vSphere.

File: docs/content/manual/advanced/addons/5846-vm-import-enhancement.md (hunk 3 above)
Comment (TachunLin) [reply docs/content/manual/advanced/addons/5846-vm-import-enhancement.md:46]: Thanks for the check. Remove the IP exposed on the check image command.

**Review by albinsun — APPROVED** [review]
LGTM

**Review by khushboo-rancher — APPROVED** [review]

**Comment by irishgordo** [comment]
Thanks for this addition @TachunLin ! :+1:

**Pull request merged**